sdk/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBSECP256K1BoolTest.scala

```diff
@@ -6,17 +6,48 @@ package speedy
 
 import com.digitalasset.daml.lf.data.support.crypto.MessageSignatureUtil
 import com.digitalasset.daml.lf.data.{Bytes, Ref}
+import com.digitalasset.daml.lf.interpretation.{Error => IE}
+import com.digitalasset.daml.lf.language.{Ast, LanguageMajorVersion}
 import com.digitalasset.daml.lf.speedy.SBuiltinFun.SBSECP256K1WithEcdsaBool
+import com.digitalasset.daml.lf.speedy.SValue.{SBool, SText}
+import com.digitalasset.daml.lf.speedy.Speedy.Control
+import org.bouncycastle.asn1.{
+  ASN1Encodable,
+  ASN1Integer,
+  ASN1ObjectIdentifier,
+  ASN1Primitive,
+  DERBitString,
+  DEROctetString,
+  DLSequence,
+}
 import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.scalacheck.{Arbitrary, Gen}
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 
+import java.math.BigInteger
 import java.security.{KeyPairGenerator, Security}
 import java.security.spec.InvalidKeySpecException
+import scala.collection.immutable.ArraySeq
+
+class SBSECP256K1BoolTestV2 extends SBSECP256K1BoolTest(LanguageMajorVersion.V2)
+
+class SBSECP256K1BoolTest(majorLanguageVersion: LanguageMajorVersion)
+    extends AnyFreeSpec
+    with Matchers
+    with ScalaCheckPropertyChecks {
+  import SBSECP256K1BoolTest._
+  import SpeedyTestLib.loggingContext
 
-class SBSECP256K1BoolTest extends AnyFreeSpec with Matchers {
   Security.addProvider(new BouncyCastleProvider)
 
+  private val compilerConfig = Compiler.Config.Default(majorLanguageVersion)
+  private val dummyMachine = Speedy.Machine.fromPureExpr(
```

Comment:
As `SBSECP256K1Bool` doesn't use its speedy machine argument, we just need some dummy machine to allow `execute` calls